docs/user/02-operation-guides/troubleshooting/03-50-recovering-from-unintentional-istio-removal.md

```diff
@@ -1,53 +1,80 @@
 # Istio unintentionally removed
-Follow the steps outlined in this troubleshooting guide if you unintentionally deleted Istio and want to restore the system to its normal state without losing any user-created resources. However, if you intended to delete Istio, the symptoms described in this document are expected and you must clean up the orphaned resources by yourself.
+Follow the steps outlined in this troubleshooting guide if you unintentionally deleted Istio and want to restore the system to its normal state without losing any user-created resources. However, if you intended to delete Istio, the symptoms described in this document are expected, and you must clean up the orphaned resources by yourself.
 
 ## Symptom
 
-* The Istio custom resource (CR) is in the `WARNING` state.
+* The Istio custom resource (CR) is in the `Warning` state.
 
 
 ### Typical log output / error messages
 
+<!-- tabs:start -->
+#### **kubectl**
+
 ```
 kubectl get istio -n kyma-system
+
 NAME      STATE
 default   Warning
 ```
 ```
 kubectl get istio default -n kyma-system -o jsonpath='{.status.description}'
+
 There are Istio resources that block deletion. Please take a look at kyma-system/istio-controller-manager logs to see more information about the warning
 ```
+#### **Kyma Dashboard**
+
+![Istio CR in the Warning state](../../../assets/istio-cr-warning-state.svg)
```

Comment:
consider SUI for this screenshot (see https://btpx.frontify.com/document/223128#/visuals/sui-graphics and https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/08-screenshots.md#simplified-user-interfaces-sui)

Reply:
Added :) thanks

Reply:
I also adjusted other screenshots. Please take a look if I did it right